examples/nodejs/publisher.js

```diff
@@ -2,17 +2,16 @@ const http = require('http');
 const querystring = require('querystring');
 
 const postData = querystring.stringify({
-    'topic': 'http://localhost:3000/demo/books/1.jsonld',
+    'topic': 'https://example.com/books/1.jsonld',
```

Comment:
Can you revert this change? Or use an env var Because currently it works "out of the box" with the demo in dev mode.